### DISCUSSION TITLE
Add ability to scale to measurements?

### DISCUSSION THREAD

**Discussion by kdm303** [OrcaSlicer/OrcaSlicer#6219] [Ideas]
I frequently have to measure object dimensions and thought it would be handy if we had ability to use the measure feature then apply the scale function to the measurement.

**Comment by ramorimdias** [comment]
It is already possible. You can just use measure toll, then edit the result. It will scale the object to match watch you entered

**Comment by kdm303** [comment]
Sorry, terrible short term memory and many distractions. :)

If I don't copy the measurement to clipboard or write it down before I exit... go back and do it again. It happens regularly.

My thought was after performing the measurement, select "scale to measurement" within the dialog, pick one or more axis and click ok.

**Reply by ramorimdias** [reply]
I don't know if are talking about the same thing, but after you use the measuring tool you can edit the value:

![image](https://github.com/user-attachments/assets/835873aa-41e7-4153-8b8c-2226bfba7588)
 
![image](https://github.com/user-attachments/assets/12879382-bbf4-4ef0-8d59-213f353e69d1)

![image](https://github.com/user-attachments/assets/ab83c0dd-045c-414c-aa6b-22ab413e85dc)

Unfortunately you cannot select one axis, but you can just go back to the scale tool and reset the modifications on the other axis.
